aptos-move/e2e-tests/goldens/language_e2e_testsuite__tests__verify_txn__test_open_publishing.exp

```diff
@@ -64,13 +64,13 @@ Ok(
                                     ],
                                 },
                                 hash: OnceCell(Uninit),
-                            }: Modification(75def505000000000100000000000000),
+                            }: Modification(661af505000000000100000000000000),
                         },
                     },
                 ),
             ),
             events: [],
-            gas_used: 650,
+            gas_used: 50841,
```

Comment:
This is expected because we set the gas unit price to 1 in this test. Shall we change it to 100 so that looks more similar to the range of numbers we had?